src/Microsoft.AspNet.Razor/Parser/TagHelpers/TagHelperBlockRewriter.cs

```diff
@@ -104,6 +107,11 @@ public static class TagHelperBlockRewriter
 
                 if (afterEquals)
                 {
+                    // We've captured all leading whitespace, the attribute name, and an equals with an optional 
+                    // quote/double quote. We're now at: " asp-for='|...'" or " asp-for=|..."
+                    // The goal here is to capture all symbols until the end of the attribute. Note this will not 
+                    // consume an ending quote due to the symbolOffset.
```

Comment:
The method itself handles a single attribute and therefore could never contain the whitespace. I don't think adding extra comment details here would add value. It might actually be confusing given the context of the method.